ui/lib/kv/addon/routes/configuration.js

```diff
@@ -13,7 +13,7 @@ export default class KvConfigurationRoute extends Route {
   model() {
     const backend = this.modelFor('application');
     return hash({
-      mountConfig: backend,
+      mountConfig: this.store.findRecord('secret-engine', backend.id),
```

Comment:
The kv Ember engine will be tackled next in the migration project so for now I just updated this to load the Ember Data model since the backend model here is now the `SecretsEngineResource`.